src/main/java/org/myrobotlab/service/Vertx.java

```diff
@@ -41,6 +43,11 @@ public class Vertx extends Service<VertxConfig> implements Gateway {
 
   private transient io.vertx.core.Vertx vertx = null;
 
+  /**
+   * If listening currently on port
+   */
+  protected boolean isListening = false;

```

Comment:
we should rename this to just "listening"

Reply:
updated